api/aperture/common/entitycache/v1/entitycache.proto

```diff
@@ -16,6 +16,7 @@ service EntityCacheService {
       get: "/v1/entity-cache/services-list"
     };
   }
+  rpc GetEntity(GetEntityRequest) returns (Entity) {}
```

Comment:
Also add GetEntites request that returns EntityCache contents (ie both maps) in a message